.cargo/audit.toml

```diff
@@ -0,0 +1,4 @@
+[advisories]
+ignore = [
+    "RUSTSEC-2023-0071", # `rsa` Marvin Attack not concerning as `rsa` is only used for tests
```

Comment:
What happens if we suddenly use it elsewhere (not in tests) ? Can we ignore it only for certain files?

Reply:
Yeah, it might happen in other repos. I'm unaware if there is an option to restrict audit ignores to specific files. Let me have a look.